bids/grabbids/config/bids.json

```diff
@@ -6,38 +6,63 @@
         {
             "name": "subject",
             "pattern": ".*sub-([a-zA-Z0-9]+)",
-            "directory": "{{root}}/{subject}"
+            "directory": "{{root}}/{subject}",
+            "aliases": ["sub"]
         },
         {
             "name": "session",
             "pattern": "ses-([a-zA-Z0-9]+)",
             "mandatory": false,
             "directory": "{{root}}/{subject}/{session}",
-            "missing_value": "ses-1"
+            "missing_value": "ses-1",
+            "aliases": ["ses"]
         },
         {
-            "name": "run",
-            "pattern": "run-(\\d+)"
+            "name": "task",
+            "pattern": "task-([a-zA-Z0-9]+)"
         },
         {
-            "name": "type",
-            "pattern": "[._]*([a-zA-Z0-9]*?)\\.[^/\\\\]+$"
+            "name": "acquisition",
+            "pattern": "acq-([a-zA-Z0-9]+)",
+            "aliases": ["acq"]
         },
         {
-            "name": "task",
-            "pattern": "task-(.*?)(?:_+)"
+            "name": "ce",
```

Comment:
"ce" is for "Contrast Enhancing agent". Not sure if there's a better primary name here.